.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: False
+
+language: python
+
+matrix:
+  include:
+    - python: 2.6
+    - python: 2.7
+    - python: 3.3
+    - python: 3.4
+
+install:
+  # Install conda
+  - wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh
+  - bash miniconda.sh -b -p $HOME/miniconda
+  - export PATH="$HOME/miniconda/bin:$PATH"
+  - conda config --set always_yes yes --set changeps1 no
+  - conda update conda
+
+  # Install dependencies
+  - conda create -n castra python=$TRAVIS_PYTHON_VERSION pytest numpy pandas
```

Comment:
yeah we have this in our repo, but it's a little old on 1.2.3 (current pip installable is 1.2.7)